fearless/Common/ViewController/NavigationController/FearlessNavigationController.swift

```diff
@@ -32,6 +32,10 @@ final class FearlessNavigationController: UINavigationController, UINavigationCo
 
         navigationBar.setBackgroundImage(UIImage(), for: UIBarMetrics.default)
         navigationBar.shadowImage = UIImage()
+        navigationBar.backIndicatorImage = R.image.iconBack()
+        navigationBar.backIndicatorTransitionMaskImage = R.image.iconBack()
+
+        navigationItem.leftItemsSupplementBackButton = true
```

Comment:
Navigation item is bonded to concrete controller. Does this work when we push new controller to the stack?

Reply:
Yes, it works like a charm
![image](https://user-images.githubusercontent.com/3176149/118985619-44260d00-b987-11eb-91bc-c58f3bdebfbc.png)
![image](https://user-images.githubusercontent.com/3176149/118985658-4e480b80-b987-11eb-82e8-17d5cb613aeb.png)

Reply:
I tried to run and don't see any difference with this line of code ```navigationItem.leftItemsSupplementBackButton = true``` and without it. As expected navigationItem of the UINavigationController shouldn't make any effect to the navigation's bar items. Can you any difference? May be I am missing something